openedx_external_enrollments/external_enrollments/icc_external_enrollment.py

```diff
@@ -186,7 +187,8 @@ def _get_icc_user_from_xml_response(self, response, method_type):
                     icc_user['username'] = field.get('VALUE')
                     break
         except Exception:  # pylint: disable=broad-except
-            pass
+            if response.get('EXCEPTION'):
```

Comment:
Shouldn't it log the exception regardless of the response? @ivanvgh

Reply:
It is not necessary at this point but I can be helpful. Already added.

Reply:
Great, thanks.